Apps/Sandcastle/gallery/iTwin Demo.html

```diff
@@ -0,0 +1,186 @@
+        const viewer = new Cesium.Viewer("cesiumContainer", {
+          terrain: Cesium.Terrain.fromWorldTerrain(),
+        });
+        const scene = viewer.scene;
+        scene.globe.show = true;
+        scene.debugShowFramesPerSecond = true;
+
+        // HTML overlay for showing feature name on mouseover
+        const nameOverlay = document.createElement("div");
+        viewer.container.appendChild(nameOverlay);
+        nameOverlay.className = "backdrop";
+        nameOverlay.style.display = "none";
+        nameOverlay.style.position = "absolute";
+        nameOverlay.style.bottom = "0";
+        nameOverlay.style.left = "0";
+        nameOverlay.style["pointer-events"] = "none";
+        nameOverlay.style.padding = "4px";
+        nameOverlay.style.backgroundColor = "black";
+        nameOverlay.style.whiteSpace = "pre-line";
+        nameOverlay.style.fontSize = "12px";
+
+        function selectFeature(feature, movement) {
+          feature.color = Cesium.Color.clone(
+            Cesium.Color.fromCssColorString("#eeff41"),
+            feature.color,
+          );
+          selectedFeature = feature;
+
+          nameOverlay.style.display = "block";
+          nameOverlay.style.bottom = `${
+            viewer.canvas.clientHeight - movement.endPosition.y
+          }px`;
+          nameOverlay.style.left = `${movement.endPosition.x}px`;
+          const element = feature.getProperty("element");
+          const subcategory = feature.getProperty("subcategory");
+          const message = `
+      Element ID: ${element}
+      Subcategory: ${classes[subcategory] || subcategory}
+      Feature ID: ${feature.featureId}`;
+          nameOverlay.textContent = message;
+        }
+
+        function unselectFeature(feature) {
+          if (!Cesium.defined(feature)) {
+            return;
+          }
+
+          feature.color = Cesium.Color.clone(Cesium.Color.WHITE, feature.color);
+          selectedFeature = undefined;
+          nameOverlay.style.display = "none";
+        }
+
+        const statusOutput = document.querySelector("#status");
+        async function init() {
+          statusOutput.innerText = "Starting export";
+
+          const start = Date.now();
+
+          statusOutput.innerText = "Creating Tileset";
+
+          let tileset = await Cesium.ITwinData.createTilesetFromModelId(
+            imodelId,
+            changesetId,
+          );
+          if (!Cesium.defined(tileset)) {
+            // TODO: this is temporary, we should not have to call the Start Export route ever after
+            // auto generation is set up
+            statusOutput.innerText = "Starting export";
+            const exportId = await Cesium.ITwinPlatform.createExportForModelId(
+              imodelId,
+              changesetId,
+            );
+            statusOutput.innerText = "Creating Tileset from export";
+            tileset = await Cesium.ITwinData.createTilesetFromExportId(exportId);
+          }
```

Comment:
I'd remove the logic to create an export. It is something we don't want to expose in our side, especially when in a few days (this week) we activate the auto-triggering for 3d tiles.
We had problems exposing this functionality in the past, so we can't expose it in this example.